### ISSUE TITLE
skip button and stop button makes a problem

### ISSUE THREAD

**Issue by KimPig** [h00z3x/Lost-Music-V2#2]
**Describe the bug**
If you press the skip or stop button, the status window does not change.

**To Reproduce**
1. Play any song
2. Press stop button
3. It changed to the first screen (No songs are playing)and it changes back to the previous song screen, and the name of the user who pressed the stop button on footer appears.

**Expected behavior**
I think It should show first screen (No songs are playing) and footer will change to the name of user.

**Comment by h00z3x** [comment]
I've tested it. 
I'll look up the skip button.
But, The stop button is ok. Since it only stops the music and doesn't disconnect the bot entirely. (but I will change it to what you want since I like it like that as well).
*Edit: It turns out sometimes the stop button resets the window and sometimes it doesn't.*

In doing so... **I've found out another bug:** *(Requires the server to be setup)*

1. Play any music.
2. Use the disconnect command.

It is supposed to change the panel but it doesn't.
I will fix it as soon as I can (I'm a little busy with studying...).
